apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -124,6 +124,7 @@ const SwapForm = () => {
     setIsSendMaxLoading,
     onNoRates: trackNoRates,
     ...locationState,
+    providers,
```

Comment:
do we need to pass `providers` on mobile to `useSwapTransaction`? At the moment that gets defaulted as an empty array, so I would have thought the mobile app won't get any exchange rates unless we pass that in

Reply:
ok, I have to add provider to llm